server/app/controllers/applicant/ApplicantProgramReviewController.java

```diff
@@ -209,6 +209,7 @@ public CompletionStage<Result> reviewWithApplicantId(
                         .setSuccessBannerMessage(flashSuccessBannerMessage)
                         .setEligibilityAlertSettings(eligibilityAlertSettings)
                         .setSummaryData(summaryData)
+                        .setProgramType(roApplicantProgramService.getProgramType())
```

Comment:
[blocking]-Should add browser tests to it?

Reply:
You're right. Added tests